packages/client/docusaurus/docs/client/02-guides/01-authentication.mdx

```diff
@@ -3,8 +3,304 @@ id: 01-authentication
+    user,
+  });
+
+  return (
+    <StreamVideo client={client}>
+      <VideoApp />
+    </StreamVideo>
+  );
+};
+```
+
+</SDKSpecific>
+
+## Anonymous users
+
+Anonymous users don't have a profile. They are not able to send audio or video, and they are not able to perform a web socket connection.
+If you try to call the `connectUser` method for an anonymous user, an error will be thrown.
+
+A typical use case for using an anonymous user are large webinars where you expect to have a lot of
+participants in the same call.
+
+However, anonymous users need a call token to join a call. Compared to the regular user tokens,
+the tokens issued for anonymous users must have additional claims and constraints:
+
+- Call tokens must have an expiration time included to avoid security problems.
+- Call tokens can contain up to 100 call ids.
+- Call tokens for anonymous users must be generated with the special `{ "user_id": "!anon" }` claim.
+- Membership / role can only be invalidated using the existing API around token invalidation (we invalidate all tokens for a user).
+- Generating a call token does not require any API interaction and can be done with any server-side SDK.
+
+Example anonymous token payload:
+
+```json
+{
+  "user_id": "!anon",
+  "exp": 1682006891, // sets the expiration date
+  "call_ids": ["default:1234", "default:5678", "audio_room:1234"] // sets the call ids that the token is valid for
+}
+```
+
+#### Connecting an anonymous user
+
+```ts
+import { StreamVideoClient, User } from '@stream-io/video-client';
+
+const user: User = {
+  id: 'anonymous-user-id',
+};
+const token = await fetchAnonToken(user);
+const videoClient = new StreamVideoClient('MY_API_KEY');
+await videoClient.connectAnonymousUser(user, token);
+
+// now you can join the call
+const call = videoClient.call('default', '1234');
+await call.join();
+```
+
+<SDKSpecific name="react">
+
+In our React Video SDK, we can use a utility hook to make this process easier.
+
+```tsx
+import { useMemo, useCallback } from 'react';
+import {
+  StreamVideo,
+  User,
+  useCreateStreamVideoClient,
+} from '@stream-io/video-react-sdk';
+
+export const App = () => {
+  const user: User = useMemo(() => ({
+    id: 'anonymous-user-id',
+  }), []);
+
+  const tokenProvider = useCallback(async () => {
+    return await fetchAnonToken(user);
+  }, [user]);
+
+  const client = useCreateStreamVideoClient({
+    apiKey: 'MY_API_KEY'
+    user,
+    tokenOrProvider: tokenProvider,
+    isAnonymous: true,
+  });
+
+  return (
+    <StreamVideo client={client}>
+      <VideoApp />
+    </StreamVideo>
+  );
+}
+```
+
+</SDKSpecific>
+
+## Guest users
+
+Guest users are users that are not registered in your system but are allowed to join a call.
+Compared to the anonymous users, guest users have a profile and can send audio, video and screenshare in the call.
+
+Additionally, the guest users can open a WebSocket connection if the right permissions are configured.
```

Comment:
Reading this line, I would like to have a link here to a guide how to set up those permissions to open a websocket for the guest user.

Reply:
Now when I'm reading this line again, I find it confusing. Permissions are/would be configurable on the backend (dashboard) and they have nothing "special" for the guest. I'm going to remove this line.